Maximilian_express_practice/controller/admin-controller.js

```diff
@@ -0,0 +1,60 @@
+const Product = require("../model/user-model");
+
+exports.postAdminproduct = (req, res) => {
+  const { username } = req.body.username;
+  const { userid } = req.body.userid;
```

Comment:
what destructuring all the variables one by one. instead of, destructure them all at once.

Reply:
Because Eslint throws the error  that's why I am destructuring the object